src/Contexts/Adaptors/SubstrateApis/bridgeTypes.json

```diff
@@ -16,6 +16,7 @@
     "id": "TokenId",
     "metadata": "Vec<u8>"
   },
-  "Address": "IndicesLookupSource",
-  "LookupSource": "IndicesLookupSource"
+  "Address": "MultiAddress",
+  "LookupSource": "MultiAddress",
+  "AccountInfo": "AccountInfoWithDualRefCount"
```

Comment:
Why do you need the last?

Reply:
I added `AccountInfoWithDualRefCount` to provide backward compatibility with the old Cere node. Without that field application returns "invalid address info" error in the console for the old node when integrated with a new polkadot.js version.